tue-dev/install.yaml

```diff
@@ -30,7 +30,7 @@
 # -------------------- ROBOTS ---------------------
 
 - type: target
-  name: ros-dashboard
+  name: dashboard
```

Comment:
What about dashboard-dev?

Reply:
A person needs to install it manually, when he/she wants to develop it.